filters/quantitative_ques/README.md

```diff
@@ -7,11 +7,7 @@ This filter is used to identify the quantitative questions
 ## Why is measuring performance on this split important?
 This filter can be used to sepearte out quantiative questions, it helps to analyse models performance on questions which require numerical understanding.It is also usefull to study possible biases in question generation.
 
-python evaluate.py -f TextContainsKeywordsFilter -p=1  
-Here is the performance of the model on the filtered set
-The accuracy on this subset which has 219 examples= 95.89041095890411
-
 ## Related Work
 
 ## What are the limitations of this filter?
-This filter serves as an example filter. It relies on lexical and purely context-free matching for filtering out examples. 
\ No newline at end of file
+This is a simple filter and separtes out questions based on some heuristics
```

Comment:
I think your earlier comment line which you removed here was actually quite useful ["lexical and context-free" matching is a clearer description of the heuristic than just saying "heuristic", and clearly specifies its behaviour]